images/opentelemetry-collector-contrib/tests/custom_config.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset -o errtrace -o pipefail -x
+
+kubectl create namespace open-telemetry-custom-config
+kubectl create configmap -n open-telemetry-custom-config custom --from-file=${PWD}/images/opentelemetry-collector-contrib/tests/custom-config.yaml
```

Comment:
in the test do we grep for some logs that custom config loaded successfully ?

Reply:
I did initially have one but then realised that by [disabling](https://github.com/chainguard-images/images/pull/1188/files#diff-4f7b53cb82ddad6410a98617178fb14e7e020c107ee7503c2992d1a1ebbbedd4R47) the auto config map means the pod will not start without the volume mount and custom config map being used.

Reply:
I'm going to resolve the comment but feel free to open it back up if you have other thoughts @ajayk.  I'm happy to grep for logs, just realised it wasn't needed while carrying out the testing locally.